pipelines/push-to-external-registry/push-to-external-registry.yaml

```diff
@@ -4,7 +4,7 @@ kind: Pipeline
 metadata:
   name: push-to-external-registry
   labels:
-    app.kubernetes.io/version: "4.1.0"
+    app.kubernetes.io/version: "4.3.0"
```

Comment:
Looks like this version number wasn't bumped properly? There's a `4.2.0` in the README changelog.